tests/automation_framework/tests/work_order_tests/test_getresult.py

```diff
@@ -0,0 +1,257 @@
+import pytest
+import logging
+import os
+import json
+from src.libs import constants
+from src.libs.avalon_test_wrapper \
+    import read_json, submit_request
+from src.libs.test_base import TestBase
+from src.utilities.verification_utils \
+    import verify_test, check_negative_test_responses
+from src.utilities.generic_utils import TestStep
+from src.utilities.generic_utils import GetResultWaitTime
+import time
+logger = logging.getLogger(__name__)
+
+
+class TestClass():
+    test_obj = TestBase()
+
+
+    @pytest.mark.work_order_get_result
+    @pytest.mark.test_work_order_get_result
+    @pytest.mark.sdk
+    def test_work_order_get_result(self):
```

Comment:
As per the naming convention we discussed, the API will be one word and no underscores needed.

Reply:
done